presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -370,4 +375,28 @@ public void testCorrelatedScalarAggregationRewriteToLeftOuterJoin()
                                                                                 project(ImmutableMap.of("NON_NULL", expression("true")),
                                                                                         node(ValuesNode.class)))))))))));
     }
+    @Test
+    public void testCountStarOverInnerAggregateQuery()
+    {
+        assertPlan(
+                "SELECT count(*) from (select sum(orderkey) from orders)",
+                output(node(ValuesNode.class).with(new ValuesMatcher(ImmutableMap.of(), Optional.of(Integer.valueOf(1)), Optional.of(ImmutableList.of(ImmutableList.of(new LongLiteral("1"))))))));
```

Comment:
use a `com.facebook.presto.sql.planner.assertions.PlanMatchPattern#values` here. Please put each node matcher in separate line